tools/c7n_gcp/c7n_gcp/resources/resourcemanager.py

```diff
@@ -29,18 +30,25 @@ class resource_type(TypeInfo):
 
 @resources.register('folder')
 class Folder(QueryResourceManager):
-
+    """GCP resource: https://cloud.google.com/resource-manager/reference/rest/v1/folders
+    """
     class resource_type(TypeInfo):
         service = 'cloudresourcemanager'
         version = 'v2'
         component = 'folders'
         scope = 'global'
+        enum_spec = ('list', 'folders', None)
         id = "name"
 
+    def get_resource_query(self):
+        if 'query' in self.data:
+            return {'parent': self.data.get('query')[0]['parent']}
```

Comment:
this should scan the array for parents, not assume the first one is the only one and has a parent key.

Reply:
Makes sense, thank you for the tip.